.github/CODEOWNERS

```diff
@@ -1 +1,3 @@
-* @linz/proj-basemaps-dev 
+* @linz/proj-basemaps-dev
+config/tileset/aerial.json @linz/proj-basemaps-dev @rclarkelinz
```

Comment:
do we need to double the devs here, I thought * would flow down to everything?

Reply:
Docs examples says:

# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.
*       @global-owner1 @global-owner2

I don't want only @rclarkelinz tagged when the aerial.json changes.

Reply:
Doesn't appear to flow down.

Reply:
TIL! thanks